conduit/data/datasets/utils.py

```diff
@@ -696,7 +696,7 @@ def stratified_split(
                     f"No samples belonging to superclass 'y={superclass}' exist in the dataset of "
                     f"type {dataset.__class__.__name__}."
                 )
-            if isinstance(value, float):
+            if isinstance(value, (float, int)):
```

Comment:
Fixing an unrelated problem that pyright is suddenly complaining about.

Reply:
how capricious